Test failure: nodeos_short_fork_take_over_test

https://github.com/AntelopeIO/spring/actions/runs/14179221445/job/39721884096

```
  File "/__w/spring/spring/build/tests/nodeos_short_fork_take_over_test.py", line 374, in <module>
    assert forkResolved, "fork was not resolved in a reasonable time. node_00 lib {} head {} node_01 lib {} head {}".format(
AssertionError: fork was not resolved in a reasonable time. node_00 lib 104 head 328 node_01 lib 116 head 236
```

`defproducerc` unexpectedly won the fork as its lib moved on its divergent block. So even though the `defproducera`/`defproducerb` fork was longer, the `defproducerc` fork won because it had the higher lib.

--- a/tests/nodeos_short_fork_take_over_test.py
+++ b/tests/nodeos_short_fork_take_over_test.py
@@ -210,9 +210,9 @@ try:
     # block number to start expecting node killed after
     preKillBlockNum=nonProdNode.getBlockNum()
     preKillBlockProducer=nonProdNode.getBlockProducerByNum(preKillBlockNum)
-    # kill at last block before defproducerl, since the block it is killed on will get propagated
+    # kill before defproducerc
     killAtProducer="defproducerb"
-    inRowCountPerProducer=12
+    inRowCountPerProducer=10 # kill before c can produce
     nonProdNode.killNodeOnProducer(producer=killAtProducer, whereInSequence=(inRowCountPerProducer-1))
 
 
@@ -303,7 +303,7 @@ try:
     killBlockNum=blockNum
     lastBlockNum=killBlockNum+(maxActiveProducers - 1)*inRowCountPerProducer+1  # allow 1st testnet group to produce just 1 more block than the 2nd
 
-    Print("Tracking the blocks from the divergence till there are 10*12 blocks on one chain and 10*12+1 on the other, from block %d to %d" % (killBlockNum, lastBlockNum))
+    Print("Tracking the blocks from the divergence till there are 2*12 blocks on one chain and 2*12+1 on the other, from block %d to %d" % (killBlockNum, lastBlockNum))
 
     for blockNum in range(killBlockNum,lastBlockNum):
         blockProducer0=prodNodes[0].getBlockProducerByNum(blockNum)

